### ISSUE TITLE
Link to User Guide more clearly from the README

### ISSUE THREAD

**Issue by borkdude** [funcool/promesa#109]
There is an excellent User Guide here: https://github.com/funcool/promesa/blob/master/doc/user-guide.md
But this is not clear from the README. I think you should link to the User Guide as one of the first things :)

**Comment by niwinz** [comment]
It is linked here:
![image](https://user-images.githubusercontent.com/843689/193464216-ee3875a6-70dc-4b15-aa1a-6712dad770ee.png)

and here:
![image](https://user-images.githubusercontent.com/843689/193464234-74b69fd1-6224-49a3-9645-d00163069fe4.png)

If you have better approach, I'm open to suggestions

**Comment by borkdude** [comment]
I mean specifically the User Guide

**Comment by niwinz** [comment]
yeah, I understand. I will put a direct link on te readme; thanks

**Comment by borkdude** [comment]
Thanks!
